src/Illuminate/Foundation/Console/ServeCommand.php

```diff
@@ -77,7 +77,17 @@ protected function serverCommand()
      */
     protected function host()
     {
-        return $this->input->getOption('host');
+        return $this->input->getOption('host') === 'network' ? $this->getLocalIp() : $this->input->getOption('host');
+    }
+
+    /**
+     * Get the user's local IP address to serve on the network.
+     *
+     * @return string
+     */
+    protected function getLocalIp()
+    {
+        return exec('ipconfig getifaddr en1') ?: '127.0.0.1';
```

Comment:
Hmm. This makes a lot of assumptions. Like, `ipconfig` exists, and the network is called `en1`.

Reply:
en1 just means wireless. the other option is en0 and it means a wired connection.

If ipconfig doesn't exist on one of those two options, then there is no local internet connection which would be infeasible for this option, so it reverts to localhost.